src/Sulu/Bundle/ContactBundle/Behat/ContactContext.php

```diff
@@ -22,12 +22,16 @@
  */
 class ContactContext extends BaseContext implements SnippetAcceptingContext
 {
+    private static $emailTypeId = 100;
```

Comment:
We have to get rid of these hardcoded ids. However, here is my recommendation on what to do with this PR. In MySQL everything works like before and regarding Postgres some important issues got removed. That's why I suggest merging this PR nonetheless to ensure that we don't drift too far off concerning postgres support. Maybe for now I'd just communicate that some bugs on postgres got fixed, after merging this PR. Full postgres support I'd communicate right after getting rid of these hardcoded id's.